### ISSUE TITLE
Remove `pages` abstraction level from code base

### ISSUE THREAD

**Issue by dmohns** [EnAccess/micropowermanager#140]
Almost all Modules are wrapped into so-called Pages. Which just adds complexity.

**Comment by munyanezaarmel** [comment]
@dmohns Awesome — this is a great refactoring! But shouldn’t we consider removing the entire pages folder, keeping only the dynamic routes and getting rid of the redundant ones?

![Image](https://github.com/user-attachments/assets/afb389af-7bd7-444a-a10c-105de77c24ca)

**Comment by dmohns** [comment]
> [@dmohns](https://github.com/dmohns) Awesome — this is a great refactoring! But shouldn’t we consider removing the entire pages folder, keeping only the dynamic routes and getting rid of the redundant ones?
> 
> ![Image](https://github.com/user-attachments/assets/afb389af-7bd7-444a-a10c-105de77c24ca)

Yes, exactly 👍 Goal of this is to entirely remove `pages` folder.

There is some added confusion because we actually have `Meter` and `Meters` (note the plural). So, in your example `Meter/_id.vue` is actually the detail page and `Meter/index.vue` the MeterList page...
